src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -150,16 +150,14 @@ const ViewResults = () => {
             </Alert.Heading>
           </Alert>
         )}
-        { /* This should just pass the isInteractiveBoxSet flag to the component */ }
-        { isInteractiveBoxSet ? (
-          <RelatedSkillsInteractiveBoxSet
+        { allowMultipleCareerInterests ? (
```

Comment:
This is the condition I was referring to in the previous comment. If we are checking for the multiple box setting here, I don't think we should have to check for it again one level deeper

Reply:
Right. I cleaned all of that up